### DISCUSSION TITLE
Fishing rod that pulls players toward you or acts as a grappling hook clinging to blocks.

### DISCUSSION THREAD

**Discussion by OmegaShaman** [PurpurMC/Purpur#757] [__Dismissed__] [answered]
The idea I have stems from some servers gamemodes (such as Hypixel) and pre-1.9 combat. The idea is that fishing rods can be used to either pull players in a certain direction or act as a grappling hook that pulls you in a certain velocity based on where the hook lands.

The config would probably have the two options that enable it, and the velocity at which a player is pulled or a player pulls on their grappling hook.

Thought it would be fun for climbing mountain biomes.

**Comment by RhythmicSys** [comment, marked as answer]
This is more of a plugin thing I feel
